src/Outsystems.SetupTools/Functions/Get-OSIntegrationStudioInfo.ps1

```diff
@@ -0,0 +1,62 @@
+function Get-OSIntegrationStudioInfo
+{
+    <#
+    .SYNOPSIS
+    Returns where the OutSystems Integration Studio install location and version.
+
+    .DESCRIPTION
+    This will returns where the OutSystems Integration Studio install location and version.
+
+    .PARAMETER MajorVersion
+    Major version. 11
+
+    .EXAMPLE
+    Get-OSIntegrationStudioInfo -MajorVersion "11"
+
+    #>
+
+    [CmdletBinding()]
+    [OutputType('Outsystems.SetupTools.IntegrationStudioInfo')]
+    param (
+        [Parameter(Mandatory = $true, HelpMessage = "11")]
```

Comment:
Why is the HelpMessage 11?

Reply:
It's the hint message for the parameter of this function (Similar to the Get-OSServiceStudioInfo)
![image](https://user-images.githubusercontent.com/88311767/183058899-e4e8a9b2-3d5b-484a-a44a-49918e6ec0ef.png)